From patches Mon Sep 17 00:00:00 2001
From: peter.crosthwaite@xilinx.com
To: peter.crosthwaite@xilinx.com, qemu-devel@nongnu.org
Cc: qemu-trivial@nongnu.org, qemu-devel@nongnu.org
Date: Fri, 19 Apr 2013 12:18:05 +1000
Message-id: 1366337885-9358-1-git-send-email-peter.crosthwaite@xilinx.com
Subject: [PATCH 1/1] m25p80: Remove bogus include of devices.h

From: Peter Crosthwaite <peter.crosthwaite@xilinx.com>

I think in the early revisions of this we had an instantiation helper
for the device in devices.h. This was later removed and this header was
left over. Removed

Signed-off-by: Peter Crosthwaite <peter.crosthwaite@xilinx.com>
---
 hw/block/m25p80.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/hw/block/m25p80.c b/hw/block/m25p80.c
index efcc7f4..b3ca19a 100644
--- a/hw/block/m25p80.c
+++ b/hw/block/m25p80.c
@@ -24,7 +24,6 @@
 #include "hw/hw.h"
 #include "sysemu/blockdev.h"
 #include "hw/ssi.h"
-#include "hw/devices.h"
 
 #ifndef M25P80_ERR_DEBUG
 #define M25P80_ERR_DEBUG 0
-- 
1.7.0.4


